numpy/testing/_private/utils.py

```diff
@@ -48,7 +48,8 @@ class KnownFailureException(Exception):
 verbose = 0
 
 IS_PYPY = platform.python_implementation() == 'PyPy'
-HAS_REFCOUNT = getattr(sys, 'getrefcount', None) is not None
+IS_PYSTON = hasattr(sys, "pyston_version_info")
```

Comment:
Why don't you use these variables in the skipif statement?

Reply:
Can we use `sys.implementation.name == 'pypy'` and `sys.implementation.name == 'pyston'` as a more straightforward way of checking implementation?

Reply:
According to  https://github.com/pyston/pyston/issues/39 the implementation.name does not seem to be an option...

Reply:
ok, thanks.